Yank/tests/test_yank.py

```diff
@@ -60,21 +56,20 @@ def test_unknown_parameters():
 @tools.raises(ValueError)
 def test_no_alchemical_atoms():
     """Test whether Yank raises exception when no alchemical atoms are specified."""
-    phase = 'solvent-implicit'
     toluene = testsystems.TolueneImplicit()
 
-    # Create parameters. All the parameters must be legal, we don't
-    # want to catch an exception different than the one we are testing.
-    phases = [phase]
-    systems = {phase: toluene.system}
-    positions = {phase: toluene.positions}
-    atom_indices = {phase: {'ligand': []}}
+    # Create parameters. With the exception of atom_indices, all other
+    # parameters must be legal, we don't want to catch an exception
+    # different than the one we are testing.
+    phase = AlchemicalPhase('solvent-implicit', '+', toluene.system, toluene.topology,
+                            toluene.positions, {'ligand': []},
+                            AbsoluteAlchemicalFactory.defaultSolventProtocolImplicit())
```

Comment:
Can you used named arguments here, just so it's super clear what arguments are what?